### PR TITLE
add_more_info_about_user

### LINKED ISSUE #6: Store more information about users
Create public profile, which could include social media usernames; favorite bands, which notes represent their favorite shows, what else?

See also #6

### PR DESCRIPTION
**Pull request by xm6264jz**
@mn4774jm I have added more information about user. Please review my work. Thanks.

### CODE REVIEW

**Comment by xm6264jz** [comment]
@mn4774jm  But I still have to work on the tests

File: lmn/views/views_users.py [hunk 1]
```diff
@@ -21,22 +21,31 @@ def my_user_profile(request):
     # TODO - editable version for logged-in user to edit their own profile
```
Comment (claraj) [inline lmn/views/views_users.py]: remove this todo comment if this is done

Comment (xm6264jz) [reply lmn/views/views_users.py]: Ok. I will do that

**Review by mn4774jm — COMMENTED** [review]
All tests are and the changes are running. I am able to add user information and it will be displayed when the user clicks on other users names. My only concerns are pretty simple. In user_profile.html the id for username notes is used twice. Pretty sure that you only really need to have it on line 54, so line 25 can be deleted.

**Comment by xm6264jz** [comment]
@mn4774jm  made the changes as you requested. deleted the username notes title

**Review by mn4774jm — APPROVED** [review]
Thanks for removing the line. It was just a little odd having a title that says user's notes twice.

**Pull request merged**